com.unity.render-pipelines.universal/Documentation~/shaders-in-universalrp.md

```diff
@@ -17,19 +17,25 @@ The Universal Render Pipeline uses a different shading approach than the Unity b
 - Autodesk Interactive Transparent
 - Autodesk Interactive Masked
 
-**Upgrade advice:** If you upgrade your current Project to URP, you can [upgrade](upgrading-your-shaders.md) built-in Shaders to the new ones. Unlit Shaders from the built-in render pipeline still work with URP.
+## Shader compatibility
 
-For [SpeedTree](https://docs.unity3d.com/Manual/SpeedTree.html) Shaders, Unity does not re-generate Materials when you re-import them, unless you click the Generate Materials or Apply & Generate Materials button.
+Lit and custom Lit shaders written for the Built-in Render Pipeline are not compatible with URP.
 
-**Note:** Unlit Shaders from the Unity built-in render pipeline work in URP.
+Unlit shaders written for the Built-in Render Pipeline are compatible with URP.
 
-## Choosing a Shader
+For information on converting shaders written for the Built-in Render Pipeline to URP shaders, see the page [Converting your shaders](upgrading-your-shaders.md).
 
-With the Universal Render Pipeline, you can have real-time lighting with either Physically Based Shaders (PBS) and non-Physically Based Rendering (PBR).
+## Choosing a shader
 
-For PBS, use the [Lit Shader](lit-shader.md). You can use it on all platforms. The Shader quality scales depending on the platform, but keeps physically based rendering on all platforms. This gives you realistic graphics across hardware. The Unity [Standard Shader](https://docs.unity3d.com/Manual/shader-StandardShader.html) and the [Standard (Specular setup) Shaders](https://docs.unity3d.com/Manual/StandardShaderMetallicVsSpecular.html) both map to the Lit Shader in URP. For a list of Shader mappings, see section [Shader mappings](upgrading-your-shaders.md#built-in-to-urp-shader-mappings).
+The Universal Render Pipeline implements Physically Based Rendering (PBR).
 
-If you are targeting less powerful devices, or your project has simpler shading, use the [Simple Lit shader](simple-lit-shader.md), which is non-PBR.
+The pipeline provides pre-built shaders that can simulate real world materials.
+
+When using PBR shaders, Materials behave consistently under different lighting conditions. PBR materials provide a set of parameters that let artists achieve consistency across hardware in Scenes with different material types.
```

Comment:
```suggestion
PBR materials provide a set of parameters that let artists achieve consistency between different material types and under different lighting conditions.
```
Reworded the sentence to remove **hardware** consistency, that is not achieved by PBR. I can try to explain a little better what I mean and you can also reword the sentence to something better. You can think of PBR as a framework that provides some parameters that model materials, these parameters are physically based. When all materials in the scene use PBR, it's easier to guarantee that they look correctly when comparing materials in a scene with each other (metals, rubber, wood, glass, car paint, hair etc) and when lighting conditions change. 

If a project doesn't use PBR it means that it relies much more on artistic direction to unify the look of these different material types.